src/util/lib/cosi/registration/graphs/index.js

```diff
@@ -404,8 +404,11 @@ class Graphs extends Registration {
         assert.equal(typeof graphIdx, 'number', 'graphIdx is required');
 
         // flat list of full metric names metric_group`metric_name (fs`/sys/fs/cgroup`df_used_percent)
-        const metrics = Object.keys(this.metrics[template.id]).map((val) => {
```

Comment:
Visibility of metrics outside the metric group / plugin.

It turns out this is only needed for variable datapoints (not variable metrics/graphs), so I reverted to the old behavior here.